bee-message/src/payload/milestone/mod.rs

```diff
@@ -217,6 +220,7 @@ pub mod dto {
                     .map_err(|_| DtoError::InvalidField("inclusionMerkleProof"))?;
                 let next_pow_score = value.next_pow_score;
                 let next_pow_score_milestone_index = value.next_pow_score_milestone_index;
+                let metadata = prefix_hex::decode(&value.metadata).map_err(|_e| DtoError::InvalidField("metadata"))?;
```

Comment:
```suggestion
                let metadata = prefix_hex::decode(&value.metadata).map_err(|_| DtoError::InvalidField("metadata"))?;
```
There's not much to criticise in this PR so you are forcing me to resort to this type of nit-picking :trollface: I think we normally just use `_` in such cases.